profiles/non_cern_machine/.setup_gemdaq.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
Is it possible the contents of this file could be moved into:

1. `.bash_profile`, and/or
2. `scripts/setup_gemdaq.sh`?

Reply:
After further investigations, this file was not used. Environment variables were either defined elsewhere or not up-to-date.